From a5f62a526738d854c1fa5d1589fec504199e2311 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Thu, 1 Sep 2022 08:56:14 +0000
Subject: [PATCH] Fix osx zmq

---
 spyder/app/start.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/spyder/app/start.py b/spyder/app/start.py
index e1286162f..e9ca84aad 100644
--- a/spyder/app/start.py
+++ b/spyder/app/start.py
@@ -10,6 +10,7 @@
 # shadowed standard libraries.
 import os
 import sys
+
 if os.environ.get('PYTHONPATH'):
     for path in os.environ['PYTHONPATH'].split(os.pathsep):
         try:
@@ -39,7 +40,8 @@
 
 # Prevent a race condition with ZMQ
 # See spyder-ide/spyder#5324.
-import zmq
+if not sys.platform == 'darwin':
+    import zmq
 
 # Load GL library to prevent segmentation faults on some Linux systems
 # See spyder-ide/spyder#3226 and spyder-ide/spyder#3332.
@@ -50,11 +52,10 @@
 
 # Local imports
 from spyder.app.cli_options import get_options
-from spyder.config.base import (get_conf_path, running_in_mac_app,
-                                reset_config_files, running_under_pytest)
-from spyder.utils.external import lockfile
+from spyder.config.base import (get_conf_path, reset_config_files,
+                                running_in_mac_app, running_under_pytest)
 from spyder.py3compat import is_unicode
-
+from spyder.utils.external import lockfile
 
 # Get argv
 if running_under_pytest():
-- 
2.34.1

