diff --git a/spyder/app/start.py b/spyder/app/start.py
index e1286162f..3a2acbede 100644
--- a/spyder/app/start.py
+++ b/spyder/app/start.py
@@ -39,7 +39,8 @@
 
 # Prevent a race condition with ZMQ
 # See spyder-ide/spyder#5324.
-import zmq
+if not sys.platform == 'darwin':
+    import zmq
 
 # Load GL library to prevent segmentation faults on some Linux systems
 # See spyder-ide/spyder#3226 and spyder-ide/spyder#3332.
